GD32F4xx/LICENSE

```diff
@@ -0,0 +1,26 @@
+   Copyright (c) 2018, GigaDevice Semiconductor Inc.
```

Comment:
Where does this file come from? HAL has copyright up to 2020, not 2018

Reply:
It's based on V2.1.0, download from Gigadevice https://www.gigadevice.com/firmware/gd32f405xxxx-firmware-library/
I will upgrade it to V2.1.3, download from http://www.gd32mcu.com/en/download/7?kw=GD32F4